benchmarks/graphics_pipeline/GraphicsBenchmarkApp.h

```diff
@@ -594,6 +594,8 @@ class GraphicsBenchmarkApp
     // UTILITY
     // =====================================================================
 
+    const Camera& GetCamera();
```

Comment:
const Camera& GetCamera() const;?

Reply:
Updated, in #391 since the commit was from that PR.